content/content-strategy/removing-content/archive-record-content/archiving-business.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: text/textblock
+section: Archiving for business records
+---
+Your agency may need to keep a record of the various versions of content. If so, you should use a recognised information management system.
+
+A content management system (CMS) or electronic document record management system (EDRMS) are the best ways of capturing your content.
+
+If your content hasn’t been captured in a recognised information management system then you'll need to make a copy some other way.
+
+## [2]If you're only making a minor change
```

Comment:
Should this be a `H3`?

Reply:
@joolswood yes this is a `H3`, as per the copy provided.

Reply:
`##` is a `H2`